src/MixedModels.jl

```diff
@@ -67,6 +67,7 @@ export @formula,
        bic,
        coef,
        coefnames,
+       coefpvalues,
        coeftable,
```

Comment:
```suggestion
```

Reply:
I don't think we want to drop `coeftable` because there is a legitimate `coeftable` method for `MixedModel`.

Reply:
Ooops. Yes, you're absolutely right.